src/controls/qml/PercentControl.qml

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) 2022 - Ed Beroset <github.com/beroset>
+ * Copyright (C) 2020 - Darrel Griët <idanlcontact@gmail.com>
+ * Copyright (C) 2015 - Florent Revest <revestflo@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import QtQuick 2.9
+import org.asteroid.controls 1.0
+
+Row {
+    id: percentControl
+    // minimum allowed value
+    property int min: 0
+    // maximum allowed value
+    property int max: 100
+    // step size per button actuation
+    property int stepsize: 10
+    // marker value appended to value display between buttons
+    property string marker: "%"
+    // initial value of the control
+    property int value: 0
+    // multiplier is the width of the label with respect to the total width
+    property real multiplier: 0.42857
+    // fontratio is the size of the font relative to the height
+    property real fontratio: 0.3
```

Comment:
fontRatio

Reply:
I changed it to `fontToHeightRatio` to make it more descriptive.